package.json

```diff
@@ -32,8 +32,8 @@
   },
   "scripts": {
     "prepublish": "babel src --out-dir dist",
-    "lint": "eslint .",
-    "tape": "babel-tape-runner 'src/**/__tests__/*.js'",
+    "lint": "eslint src",
```

Comment:
On windows I think it tries to lint the `dist` directory as well. Does it do that on *nix too? I think because the code in the `dist` directory is generated by babel it doesn't match the styleguide and the linter fails it.